packages/flutter/test/material/raw_material_button_test.dart

```diff
@@ -52,23 +52,25 @@ void main() {
     expect(semantics, hasSemantics(
       TestSemantics.root(
         children: <TestSemantics>[
-        TestSemantics(
-          id: 1,
-          flags: <SemanticsFlag>[
-            SemanticsFlag.isButton,
-            SemanticsFlag.hasEnabledState,
-            SemanticsFlag.isEnabled,
-          ],
-          actions: <SemanticsAction>[
-            SemanticsAction.tap,
-          ],
-          label: '+',
-          textDirection: TextDirection.ltr,
-          rect: const Rect.fromLTRB(0.0, 0.0, 48.0, 48.0),
-          children: <TestSemantics>[],
-        ),
-      ]
-    ), ignoreTransform: true));
+          TestSemantics(
+            id: 1,
+            flags: <SemanticsFlag>[
+              SemanticsFlag.isButton,
+              SemanticsFlag.hasEnabledState,
+              SemanticsFlag.isEnabled,
+            ],
+            actions: <SemanticsAction>[
+              SemanticsAction.tap,
+            ],
+            label: '+',
+            textDirection: TextDirection.ltr,
+            rect: const Rect.fromLTRB(0.0, 0.0, 48.0, 48.0),
+            children: <TestSemantics>[],
+          ),
+        ]
```

Comment:
nit: should there be a trailing comma here?

Reply:
I have a rule to lint missing trailing commas but (for now) I made an exception to make it only lint missing trailing commas if the argument list has a length > 1. That's why there's still a missing trailing comma here.

This exception is here to allow things like:

```dart
          information.add(ErrorDescription(
            'The test API method "${originalGuarder.methodName}" '
            'from class ${originalGuarder.className} '
            'was called from ${originalGuarder.callerFile} '
            'on line ${originalGuarder.callerLine}, '
            'but never completed before its parent scope closed.'   // there should be a trailing comma here
          ));
```

From dartfmt POV there should be a trailing comma to keep this kind of formatting. Perhaps/eventually the rule should remove the exception. WDYT?